test/AppTest/Handler/HomePageHandlerFactoryTest.php

```diff
@@ -46,7 +46,7 @@ public function testFactoryWithTemplate()
 
         $factory = new HomePageHandlerFactory();
 
-        $homePage = $factory($this->container->reveal(), null, get_class($this->container->reveal()));
+        $homePage = $factory($this->container->reveal(), null, HomePageHandler::class);
```

Comment:
Not sure if we need it at all, as the current implementation uses only 1 parameter:
https://github.com/zendframework/zend-expressive-skeleton/blob/master/src/App/src/Handler/HomePageHandlerFactory.php#L14

Reply:
I know. But the way it's used is wrong. So it needs to be fixed or removed. I wanted to remove it but I have no idea why it's there.

Reply:
The class name should be the second parameter, not the third: https://github.com/zendframework/zend-servicemanager/blob/master/src/Factory/FactoryInterface.php#L37

This looks like the factories are written incorrectly.